doc/build/dts/dt-vs-kconfig.rst

```diff
@@ -69,3 +69,41 @@ There are **exceptions** to these rules:
 * Devicetree's ``chosen`` keyword, which allows the user to select a specific
   instance of a hardware device to be used for a particular purpose. An example
   of this is selecting a particular UART for use as the system's console.
+
+Automatic Kconfig symbols from devicetree
+*****************************************
+
+During the devicetree processing step CMake runs
+:zephyr_file:`scripts/dts/gen_driver_kconfig_dts.py`, which scans every binding
+under :file:`dts/bindings` and writes ``Kconfig.dts`` into the build's
```

Comment:
should probably mention it's in dts roots, including the zephyr one, since modules can add them and users on the command line